task/rhacs-image-check/3.71/README.md

```diff
@@ -0,0 +1,63 @@
+# Red Hat Advanced Cluster Security Image Check Task
+
+Check an image against RHACS build and deploy lifecycle policies to validate a pipeline run using `roxctl`.
+
+It's a companion to the rhacs-image-scan task, which returns full vulnerability scan results for an image.
+
+## Prerequisites
```

Comment:
@dlbewley, the following Task already exists in the catalog with the name [stackrox-image-check](https://github.com/tektoncd/catalog/tree/main/task/stackrox-image-check/0.1) (I guess). If so, can we please bump the version of the existing task if there are any changes?

Reply:
@vinamra28 these tasks are developed specifically to target RHACS installations rather than the upstream solution. i am amenable to contributing enhancements to those tasks as appropriate in a subsequent PR.